frontend/src/components/AppPercentage.vue

```diff
@@ -3,13 +3,29 @@
 -->
 
 <template>
-  <div class="ring">
-    <div>{{ score.toFixed(1) }}</div>
+  <div v-if="props.type === `ring`" class="ring">
+    <b>{{ score.toFixed(1) }}</b>
     <svg viewBox="-50 -50 100 100">
       <circle cx="0" cy="0" r="50" />
       <path :d="d" />
     </svg>
   </div>
+
+  <div v-else class="bar">
+    <svg viewBox="-50 -50 100 30">
+      <rect x="-50" y="-50" rx="3" width="100%" height="100%" fill="none" />
+      <rect
+        x="-49"
+        y="-49"
+        rx="3"
+        :width="fillWidth"
+        height="95%"
+        fill="none"
+        class="fill"
+      />
+    </svg>
+    <b>{{ `${(score * 100).toFixed(0)}%` }}</b>
```

Comment:
Since we have a bunch of different metrics/measures and places where we might use this component, I'm envisioning it's probably better to just get rid of the `score` prop all together and just use a `<slot>` here, and let the consuming component define number of decimal places, "%", etc.